modules/openapi-generator/src/main/resources/dart2/serialization/native/native_class.mustache

```diff
@@ -1,21 +1,40 @@
 class {{{classname}}} {
 {{>dart_constructor}}
-
-  {{#vars}}
+{{#vars}}
   {{#description}}
   /// {{{.}}}
   {{/description}}
   {{^isEnum}}
     {{#minimum}}
-  // minimum: {{{.}}}
+      {{#description}}
+  ///
+      {{/description}}
+  /// Minimum value: {{{.}}}
     {{/minimum}}
     {{#maximum}}
-  // maximum: {{{.}}}
+      {{#description}}
+        {{^minimum}}
+  ///
+        {{/minimum}}
+      {{/description}}
+  /// Maximum value: {{{.}}}
     {{/maximum}}
+    {{^isNullable}}
+      {{^required}}
+        {{^defaultValue}}
+  ///
+  /// Please note: This property should have been non-nullable! Since the specification file
+  /// does not include a default value (using the "default:" property), however, the generated
+  /// source code must fall back to having a nullable type.
+  /// Consider adding a "default:" property in the specification file to hide this note.
+  ///
+        {{/defaultValue}}
+      {{/required}}
+    {{/isNullable}}
   {{/isEnum}}
-  {{{datatypeWithEnum}}} {{{name}}};
+  {{{datatypeWithEnum}}}{{#isNullable}}?{{/isNullable}}{{^isNullable}}{{^required}}{{^defaultValue}}?{{/defaultValue}}{{/required}}{{/isNullable}} {{{name}}};
```

Comment:
A model property can be null in 2 cases:

1) The spec. file specifies that it's nullable.
2) The spec. file specifies that it's non-nullable, but doesn't specify either a default value or a `required` value.

This check is replicated throughout this Mustache template.